src/bp-core/classes/class-bp-invitation-manager.php

```diff
@@ -378,6 +378,18 @@ public function get_invitations( $args = array() ) {
 		// Use the class_name property value.
 		$args['class'] = $this->class_name;
 
+		/**
+		 * We are making sure that we are passing limits and page
+		 * @since 1.4.7
+		 */
+		$page = 1;
+		if (isset($_POST['page'])){
+			$page = (int) sanitize_text_field($_POST['page']);
+		}
+
+		$args['page'] = $page;
+		$args['per_page'] = 100;
```

Comment:
@shohelbuddyboss `per_page`  should be 20 by default.

Reply:
And if we are limiting the requests on the page, did we load others via pagination or load more?

Reply:
* fixed space issue.
* per page will be load given amount of pending invitation at a time in, before it was no limit, so PHP execution time issue we saw. It's I increased to 500, so at a time, 500 pending requests will be load at a time and auto load more pagination can pull the results from 500 after 500 invitation done, users need to reload again, (Facebook also doing this)

Reply:
@shohelbuddyboss Who will tell users to reload again?